scripts/microshift-cleanup-data.sh

```diff
@@ -29,11 +32,13 @@ function stop_disable_services() {
     for service in microshift microshift-etcd ; do
         systemctl stop --now   ${service} 2>/dev/null || true
         systemctl reset-failed ${service} 2>/dev/null || true
+        SERVICE_STOPPED=true
```

Comment:
Why not just echo here? That way if something goes wrong and the script exits there is still information about what progress it did make before the error.

Reply:
Well, we already have a message about stopping MicroShift. I think the ask was to have it towards the end where users can see it.

Reply:
@dhensel-rh , what's your preference? I can add the message right on the spot vs in the end.

Reply:
@ggiguash I guess my request was alerting the user that MicroShift would need to be restarted at the bottom of the script. I guess it is implied at the top when the first line says Stopping MicroShift services. Sorry if it feels I am being pedantic

Reply:
Does the current messaging address this request? We're printing the following in the end of the output.
```
...
...
MicroShift service was stopped
MicroShift service was disabled
Cleanup succeeded
```

Reply:
I kind of agree with @dhensel-rh. We cannot assume that someone using that script lives and breathes systemd, so adding some friendly UX might be good

Reply:
@ggiguash Yes, your message addresses the request.